packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -38,16 +96,26 @@ export const CurrencyToSend = () => {
         </Typography>
         <Block display="flex" marginTop={5} marginBottom={1}>
```

Comment:
You should wrap all fields elements in a Form, which is also available in medulas, doing this, you should implement validation in the same way as we are doing, in the same package as in medulas you can see a story, or directly find forms in the extension.